data/common.yaml

```diff
@@ -0,0 +1 @@
+--- {}
```

Comment:
at vox pupuli we avoid empty hiera files, I'm not sure if CAT/Puppet has a best practice here.